python/src/nanoarrow/_lib.pyx

```diff
@@ -790,8 +1043,6 @@ cdef class CArray:
             respectively.
         """
         self._assert_valid()
-        if requested_schema is not None:
-            raise NotImplementedError("requested_schema")
```

Comment:
Is there a reason this was removed? (because it's still not supported I think, the keyword is just ignored otherwise)